### PR TITLE
Update task to populate Prince George CASA details

### LINKED ISSUE #831: casa_case becomes transition_aged_youth when they turn 14
**Description**
On the first day of a `casa_case`'s birth month in the year they turn 14 (`birth_month_year_youth` in the `casa_cases` table), 
`transition_aged_youth` should change from `false` to `true`. 

**Acceptance criteria**
- [ ] Backend: Under the conditions listed above, `transition_aged_youth` automatically becomes `true`
- [ ] Frontend: When `transition_aged_youth` is `true`, 🐛 🦋  appears on the front end with the text `Yes`

Screen Shot 2020-09-30 at 8 06 34 PM (2)

### PR DESCRIPTION
**Pull request by jmkoni**
### What github issue is this PR for, if any?
Resolves #831 

### What changed, and why?
- made task work with empty DB or populated DB
- added tests
  - one of the tests had to be commented out because AfterParty is only allowing the task to be run once
  - each passes if run in isolation, but either will fail if the other test is run first

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
Added tests, had to comment one out (see screenshot to see both passing separately)

### Screenshots please :)
Screen Shot 2020-10-05 at 9 11 23 AM

### CODE REVIEW

**Review by littleforest — CHANGES_REQUESTED** [review]
@jmkoni there's a weird thing about running rake tasks multiple times -- you have to reenable it. See [this post](https://til.hashrocket.com/posts/4897ed42b7-invoking-rake-tasks-multiple-times) for an example. So you should be able to remove the `unless Rails.env.test?` exclusion on line 27 of the after party task, and then hopefully be able to remove the `xit` from the second test.

**Comment by jmkoni** [comment]
@littleforest do you mean `Casa::Application.load_tasks`?

**Comment by littleforest** [comment]
@jmkoni no, you will need to make this change:

```
Rake::Task["after_party:populate_prince_george_casa_org_details"].invoke
Rake::Task["after_party:populate_prince_george_casa_org_details"].reenable
```

File: lib/tasks/deployment/20200907142217_populate_prince_george_casa_org_details.rake [hunk 1]
```diff
@@ -5,29 +5,28 @@ namespace :after_party do
...
+                     footer_links: [
+                       ["https://pgcasa.org/contact/", "Contact Us"],
+                       ["https://pgcasa.org/subscribe-to-newsletter/", "Subscribe to newsletter"],
+                       ["https://www.givedirect.org/give/givefrm.asp?CID=4450", "Donate"]
+                     ])
     # Update task as completed.  If you remove the line below, the task will
     # run with every deploy (or every time you call after_party:run).
-    AfterParty::TaskRecord
-      .create version: AfterParty::TaskRecorder.new(__FILE__).timestamp
+    unless Rails.env.test?
```
Comment (littleforest) [inline lib/tasks/deployment/20200907142217_populate_prince_george_casa_org_details.rake:27]: @jmkoni can you remove this line now that you added `reenable`?

File: spec/lib/tasks/deployment/populate_prince_george_casa_org_details_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    aggregate_failures do
+      expect(casa_org.display_name).to eq("CASA / Prince George's County, MD")
+      expect(casa_org.address).to eq("6811 Kenilworth Avenue, Suite 402 Riverdale, MD 20737")
+      expect(casa_org.footer_links.length).to eq 3
+      expect(casa_org.casa_org_logo.alt_text).to eq "CASA Logo"
+    end
+  end
+
+  # Both of these pass if run separately, but something about AfterParty
+  # (I think) is causing the task to not be run the second time
```
Comment (littleforest) [inline spec/lib/tasks/deployment/populate_prince_george_casa_org_details_spec.rb:19]: Also can remove these comments now!

**Review by littleforest — APPROVED** [review]
🎉 👯‍♀️

**Pull request merged**